docs/quick_start.rst

```diff
@@ -226,179 +368,245 @@ A user-supplied parsing function is employed to parse the output files.
    :lines: 7-24
    :language: python
 
+Run this Python script to evaluate the experiments:
+
+.. code-block:: bash
+
+  $ python3 eval.py
+
+Output:
+
+::
+
+                  comparisons      swaps      time
+  experiment                                      
+  bubble-sort        405812.5  205647.25  0.043447
+  insertion-sort     196834.5  208978.00  0.021463
+
 Managing Instances
 ------------------
-Before launching the experiments, make sure that all your instances are available.
-Instances can be checked with:
+
+Before launching the experiments, we neede to make sure that all our instances
```

Comment:
There is an "e" too much in "neede": "..., we need to make sure that ..."

Reply:
Thanks, done.